<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>manager</class>
 <widget class="QMainWindow" name="manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(228, 27, 27);
image: url(:/new/prefix1/image/menu6.jpg);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="main_tab">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1024</width>
      <height>600</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::TabFocus</enum>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::West</enum>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <property name="movable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string/>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>834</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(217, 13, 13);
image: url(:/new/prefix1/image/menu63.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="attance_tw">
      <property name="geometry">
       <rect>
        <x>23</x>
        <y>100</y>
        <width>791</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="return_button_1">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_1">
      <property name="geometry">
       <rect>
        <x>29</x>
        <y>58</y>
        <width>170</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;Ubuntu&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>1001</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1002</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1003</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1004</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="check_bt">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>54</y>
        <width>51</width>
        <height>37</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="all_bt">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>56</y>
        <width>100</width>
        <height>39</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>834</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(239, 0, 0);
image: url(:/new/prefix1/image/menu61.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTableWidget" name="visit_tw">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>100</y>
        <width>801</width>
        <height>381</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="return_button_2">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_6">
      <property name="geometry">
       <rect>
        <x>28</x>
        <y>58</y>
        <width>170</width>
        <height>37</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 15pt &quot;Ubuntu&quot;;</string>
      </property>
      <item>
       <property name="text">
        <string>1001</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1002</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1003</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1004</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="check_bt_2">
      <property name="geometry">
       <rect>
        <x>199</x>
        <y>59</y>
        <width>51</width>
        <height>37</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string/>
     </attribute>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>-10</y>
        <width>834</width>
        <height>600</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(244, 64, 64);
image: url(:/new/prefix1/image/menu62.jpg);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>167</x>
        <y>130</y>
        <width>280</width>
        <height>29</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>123</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1234</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>12345</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>123456</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>0123456</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_2">
      <property name="geometry">
       <rect>
        <x>167</x>
        <y>170</y>
        <width>280</width>
        <height>29</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>37</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>37.1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>37.2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>37.3</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_3">
      <property name="geometry">
       <rect>
        <x>167</x>
        <y>210</y>
        <width>280</width>
        <height>29</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>8:30</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8:40</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>8:50</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>9:00</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>167</x>
        <y>250</y>
        <width>280</width>
        <height>29</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>192.168.1.111</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.1.112</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>192.168.1.113</string>
       </property>
      </item>
     </widget>
     <widget class="QComboBox" name="comboBox_5">
      <property name="geometry">
       <rect>
        <x>167</x>
        <y>290</y>
        <width>280</width>
        <height>29</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>60</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>70</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>80</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="return_button_3">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>50</y>
        <width>51</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
